### DISCUSSION TITLE
使用validateConnection如何高亮不能被连接的节点

### DISCUSSION THREAD

**Discussion by nomis-dev** [antvis/X6#2204] [Q&A]
拖动边，鼠标移到节点上，如果这个节点不能被连接如果高亮节点。
node:mouseenter/mouseleave事件在拖动边的时候好像不会触发。

**Comment by NewByVector** [comment]
不能被连接为什么还要高亮呀？

**Reply by nomis-dev** [reply]
抱歉太久没看github，项目是用react自定义节点实现，已经通过validate时候修改data改变classname方式实现
